app/scripts/services/images.js

```diff
@@ -229,7 +229,7 @@ angular.module("openshiftConsole")
                 var value = "";
                 if (ind > 0) {
                   key = entry.substring(0, ind);
-                  if (ind + 1 < entry.length) {
+                  if (ind + 1 < _.size(entry)) {
```

Comment:
this is a string not an array, i don't mind the change for the sake of getting these PRs in, but for future reference I don't think its really necessary to eliminate length checks on strings